pos_accounting/src/pool/delta/operator_impls.rs

```diff
@@ -186,11 +186,11 @@ impl<'a> PoSAccountingOperatorWrite for PoSAccountingDelta<'a> {
     }
 }
 
-impl<'a> PoSAccountingDelta<'a> {
+impl<P: PoSAccountingView> PoSAccountingDelta<P> {
     fn undo_create_pool(&mut self, undo: CreatePoolUndo) -> Result<(), Error> {
         let (pledge_amount, undo_data) = match undo.data_undo {
             PoolDataUndo::DataDelta(v) => *v,
-            PoolDataUndo::Data(_) => unreachable!("incompatible PoolDataUndo supplied"),
+            PoolDataUndo::Data(_) => panic!("incompatible PoolDataUndo supplied"),
```

Comment:
Why changing to panic?

Reply:
The idea was that this code is reachable if someone passes the argument with the wrong type. Maybe this should be an error